content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,55 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
 weight: 300
 toc: true
 ---
+
+Many of the rules for how **Asset Processor** will process content for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
+
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to setup and deploy to additional platforms.
+
+By default, the Asset Processor will create a worker job for each logical core on the host machine. If you wish to manage this yourself, you can adjust the setting `Amazon/AssetProcessor/Settings/Jobs/maxJobs`. A value of zero will have the Asset Processor create a job for each logical core on your machine. A value of one can be useful when debugging, if you wish to attach a debugger to a running builder processing a specific asset. Other values lower than your logical core count can be useful to manage the impact asset processing has on your machine while you are trying to perform other tasks, with the tradeoff of a longer time before all assets are processed.
+
+The asset cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. Documentation for this feature is still in progress, feel free to reach out on Discord if you have questions about the Asset Cache Server in the meantime.
```

Comment:
```suggestion
The asset cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`.
```

Rather than directing users to the Discord, file a GitHub issue requesting documentation for this feature. If you want a reference to the requirement improvement in the document itself, use the `{{< todo issue="###" >}}` shortcode (see [smoketest](https://github.com/o3de/o3de.org/blob/main/content/smoketest.md) for an example). 

Also even if this isn't completely documented, include the required formatting (I'm assuming `<ipv4-address>:<port>` ?)

Reply:
It's not just documentation we need for it, if someone wants to use this they pretty much have to talk to us directly. Cleaning up this feature is on my team's backlog, but right now you can't really use the feature without having someone on my team walk you through it. It's a 1.x feature that was built for some specific use cases, we want to expand it to support more teams but we haven't gotten around to it.

Reply:
I've added a note on the format. It's not an ip address, it's a network share address.